packages/codemods/README.md

```diff
@@ -0,0 +1,83 @@
+# ts-morphin-migration
+
+ts-morphin-migration is a package created and managed with the `create-just` utility.
+
+## Next Steps
+
+Run
+
+```sh
+yarn
+yarn start-test
+```
+
+To start testing the codemods
+
+## To add a codemod
+
+Add your codemods to the `./src/mods` folder with `.mod.ts|tsx` as the file type.
+
+## Test with the actual package:
+
+Run
+
+```sh
+yarn build
+```
+
+To build the upgrade.js file
+
+Run
+
+```sh
+npm pack
+```
+
+from the codemods root to create a tar file for testing. Move the created tar file to the package you want to test and run
+
+```sh
+npx <tarFileName>
+```
+
+## Executing a codemod (Prototype)
+
+Make sure that fluent is installed in your package.
```

Comment:
```suggestion
Make sure that Fluent UI is installed in your package.
```

Reply:
Even then, I'm confused. What does this mean? In your "application"? What "Fluent UI" package? Why is this package separate from other packages?

E.g. if `react-button` wants to ship a mod with the package, can the mod live with the package? What pattern are we suggesting to use for partners (like react-date-time) that want to write codemods? If they are separate from the package they apply to, how are they locked to the package they apply to (how does versioning work?)

Reply:
Right now, for v1, this will run all codemods present in this package. There is no reason we can't change where the execution file looks to get codemods but that can be a feature down the line.